src/utils/sortByAddress.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from 'viem';
+
+interface ObjectWithAddress {
+    address: Address;
+}
+// Sorts any array of objects by an 'address' param
+export function sortByAddress<T extends ObjectWithAddress>(
+    objectWithAddressArray: T[],
+): T[] {
+    return objectWithAddressArray.sort((a, b) =>
+        a.address.localeCompare(b.address),
```

Comment:
I'm not familiar with localCompare - are we sure it sorts as we expect?

Reply:
This function compares strings using the Unicode charset:

`0, 1, 2, ... 9, A, B, C, D, ... Z, a, b, c, ..., z`

 It works fine if both strings are lowercase, but we would not have the correct result comparing a "0xE5..." with a "0xe4..." address, thanks for noticing, I'll add an extra step to convert both strings to lowercase